kodev build fails under Fedora 26

* KOReader version: master

#### Issue
In the process of creating a plugin (to be able to disable touch with a physical button on the inkbook prime, stay tuned), I wanted to build koreader but failed to do so.

I'm running Fedora 26 and think that I do have all necessary requirements (lua, nasm, ...)

This is what I did:
```
git clone https://github.com/koreader/koreader.git
cd koreader
./kodev build
```

[And here is the output from `./kodev build -v`](https://gist.github.com/silberzwiebel/9db20ff2f2f29a5cc1f319c2021def02). It fails at configuring djvulibre. Manually going into that directory and doing `.\configure && make` works, though.

> But you're conspicuously leaving out all of the environment variables in that case. ;-) 

Sorry, I did not do this intentionally. To me those files often look more like noise than like signal.
Installing  `libstdc++-static` indeed did fix this problem but I ran into another one:

```
[ 12%] Performing build step for 'turbo'
cd /home/thomas/zeugs/fossgits/mykoreader/base/thirdparty/turbo/build/x86_64-redhat-linux/turbo-prefix/src/turbo && sh -c "make CC=\"/usr/bin/ccache gcc -O2 -ffast-math -pipe -fomit-frame-pointer  -march=native -fPIC -I/home/thomas/zeugs/fossgits/mykoreader/base/thirdparty/openssl/build/x86_64-redhat-linux/openssl-prefix/src/openssl/include\" LDFLAGS=\"-Wl,-O1 -Wl,--as-needed -static-libstdc++ -lcrypto -lssl 		 		 		-L/home/thomas/zeugs/fossgits/mykoreader/base/thirdparty/openssl/build/x86_64-redhat-linux/openssl-prefix/src/openssl -Wl,-rpath,'\\\$\\\$ORIGIN/../libs'\" all"
make[5]: Verzeichnis „/home/thomas/zeugs/fossgits/mykoreader/base/thirdparty/turbo/build/x86_64-redhat-linux/turbo-prefix/src/turbo“ wird betreten
make -C deps/http-parser library
make[6]: Verzeichnis „/home/thomas/zeugs/fossgits/mykoreader/base/thirdparty/turbo/build/x86_64-redhat-linux/turbo-prefix/src/turbo/deps/http-parser“ wird betreten
/usr/bin/ccache gcc -O2 -ffast-math -pipe -fomit-frame-pointer  -march=native -fPIC -I/home/thomas/zeugs/fossgits/mykoreader/base/thirdparty/openssl/build/x86_64-redhat-linux/openssl-prefix/src/openssl/include  -I. -fPIC -DHTTP_PARSER_STRICT=0  -O3  -Wall -Wextra -Werror -c http_parser.c -o libhttp_parser.o
http_parser.c: In Funktion »http_parser_parse_url«:
http_parser.c:2096:18: Fehler: diese Anweisung könnte durchfallen [-Werror=implicit-fallthrough=]
         found_at = 1;
         ~~~~~~~~~^~~
http_parser.c:2099:7: Anmerkung: hier
       case s_req_server:
       ^~~~
cc1: Alle Warnungen werden als Fehler behandelt
make[6]: *** [Makefile:82: libhttp_parser.o] Fehler 1
```

The error translates to: "All warnings are treated as errors". I manually deleted the string `-Werror` from the file `koreader/base/thirdparty/turbo/build/x86_64-redhat-linux/turbo-prefix/src/turbo/deps/http-parser/Makefile` but I guess this is not the way to go ;)
But probably this is an issue of the `turbo` lib then?

Anyway, after doing this, `./kodev build` is succesful. However, `.\kodev run` won't start due to missing translation files (maybe need to add this to the readme?). Pulling the translation as described in `l10n/README.md`, however, does not work:

```
tx pull -a -f
New translations found for the following languages: el, zh, fi_FI, vi, ca, it_IT, ug, vi_VN, cs, nl_NL, eu, gl, es, ru, tr, pt_PT, pl, bg_BG, fr, zh_TW, pt_BR, bn, ar_AA, hu, fa, ko_KR, cs_CZ, ja, zh_CN, kk, sv, de, zh_TW.Big5, uk, pl_PL, nb_NO
Pulling new translations for resource koreader.koreader (source: templates/koreader.pot)
 -> el: el/koreader.po
DecodeError: ('Received response with content-encoding: gzip, but failed to decode it.', error('Error -2 while decompressing: inconsistent stream state',))
make: *** [Makefile:9: pull] Fehler 1
```

--- a/README.md
+++ b/README.md
@@ -80,6 +80,9 @@ sudo apt-get install build-essential git patch wget unzip \
 gettext autoconf automake cmake libtool nasm luarocks \
 libssl-dev libffi-dev libsdl2-dev libc6-dev-i386 xutils-dev linux-libc-dev:i386 zlib1g:i386
 ```
+
+If you are running Fedora, be sure to install the package `libstdc++-static`.
+
 That's all you need to get the emulator up and running with `./kodev build` and `./kodev run`.
 
 Cross compile toolchains are available for Ubuntu users through these commands:

